neuro_extras/main.py

```diff
@@ -222,7 +222,7 @@ async def _create_copier_container(
             args = f"-x {args}"
         return neuro_api.Container(
             image=neuro_api.RemoteImage.new_external_image("neuromation/neuro-extras"),
-            resources=neuro_api.Resources(cpu=4.0, memory_mb=4096),
+            resources=neuro_api.Resources(cpu=2.0, memory_mb=4096),
```

Comment:
Why do we tend to save CPU here?

1. I understand that there should be preset not exact resources definition, but AFAIK we follow preset definition on `neuro-public`: `cpu-small        1    4.0G` (Jobs Available: 205). If we change only cpu, then we just waste resources.
2. Are we sure tar/unzip/gunzip/rsync don't use parallelism?

Reply:
1. Sorry, I don't get why we waste resources. Could you please elaborate?
2. I'm sure pure tar and rsync are not CPU-bound in all cases. zip/gzip might be, but I would imagine there're also more IO-bound on our days. Not sure though.